activerecord/test/schema/schema.rb

```diff
@@ -503,6 +503,20 @@
     t.column :weight, :integer
   end
 
+  if supports_partitioned_indexes?
```

Comment:
:thinking: I think you can move this to `test/schema/postgresql_specific_schema.rb`.